### PR TITLE
rlang update

### PR DESCRIPTION
**Pull request by graemeblair**

### CODE REVIEW

File: R/declare_assignment.R
Comment (nfultz) [inline R/declare_assignment.R:18]: Maybe delete this line? Usually I try to keep the importFroms near the functions that actually use them.

**Comment by graemeblair** [comment]
@nfultz will wait for your signoff before merging in *eek*

**Review by nfultz — APPROVED** [review]
This looks OK as long as the tests are passing.

**Pull request merged**
